otherlibs/runtime_events/runtime_events_consumer.c

```diff
@@ -1334,3 +1334,11 @@ CAMLprim value caml_ml_runtime_events_read_poll(value wrapper,
 
   CAMLreturn(Val_int(events_consumed));
 }
+
+CAMLprim uint64_t caml_ml_runtime_current_timestamp(value unit) {
+  return caml_time_counter();
+}
+
+CAMLprim value caml_ml_runtime_current_timestamp_byte(value unit) {
+  return caml_copy_int64(caml_time_counter());
+}
```

Comment:
Very nitpicky: the convention I'm familiar with in the runtime is to use `caml_foo` for the value-returning version (your `_byte` variant) and then `caml_foo_unboxed` for the unboxed version. I would rather stick to this.

Reply:
done.